src/main/java/foxtrot/uniform/charlie/kilo/model/User.java

```diff
@@ -1,21 +1,29 @@
 package foxtrot.uniform.charlie.kilo.model;
 
+import java.util.Collection;
+import java.util.List;
 import javax.persistence.Column;
 import javax.persistence.Entity;
+import javax.persistence.FetchType;
 import javax.persistence.GeneratedValue;
 import javax.persistence.GenerationType;
 import javax.persistence.Id;
+import javax.persistence.ManyToMany;
 import javax.persistence.Table;
+import org.springframework.security.core.GrantedAuthority;
+import org.springframework.security.core.userdetails.UserDetails;
 
 @Entity
 @Table(name = "users")
-public class User {
+public class User implements UserDetails {
     @Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
     @Column(unique = true)
     private String email;
     private String password;
+    @ManyToMany(fetch = FetchType.EAGER)
+    private List<Role> roles;
```

Comment:
it makes more sense to have roles stored in set, they should not repeat themselves